packages/core/vue/src/useWallet.ts

```diff
@@ -85,22 +83,19 @@ export const createWalletStore = ({
 
     // Helper methods to set and reset the main state variables.
     const setState = (state: {
-        wallet: Wallet | null;
-        adapter: Adapter | null;
+        wallet: Adapter | null;
         publicKey: PublicKey | null;
         ready: boolean;
         connected: boolean;
     }) => {
         wallet.value = state.wallet;
-        adapter.value = state.adapter;
         ready.value = state.ready;
         publicKey.value = state.publicKey;
         connected.value = state.connected;
     };
     const resetState = () => {
         setState({
             wallet: null,
-            adapter: null,
             ready: false,
```

Comment:
This will need to change too. The ready state is no longer binary.

Reply:
Got it, I'll switch it to a WalletReadyState. One thing I was wondering was whether the default value for that should be WalletReadyState.NotDetected or WalletReadyState.Loadable. I can't really see when one is used over another from looking at the Phantom and WalletConnect implementations. The [Phantom implementation defaults to 'NotDetected'](https://github.com/solana-labs/wallet-adapter/blob/master/packages/wallets/phantom/src/adapter.ts#L62-L65) when not supported while the [WalletConnect implementation defaults to 'Loadable'](https://github.com/solana-labs/wallet-adapter/blob/master/packages/wallets/walletconnect/src/adapter.ts#L49-L50).

Reply:
Ah, the usual default is NotDetected. Loadable is only for a few adapters that load a third party library instead of attaching to a browser extension.

The distinction here is that for loadable wallets there nothing to detect (they can always be loaded).